clouddrift/adapters/gdp1h.py

```diff
@@ -79,8 +80,8 @@ def download(
     os.makedirs(tmp_path, exist_ok=True)
 
     if url == GDP_DATA_URL:
```

Comment:
This is not useful anymore but should we keep it to bring awareness of some possible changes upstream?

Reply:
I kept it in case eventually there is a difference with the two datasets. If we are sure it is not going to change I would remove it.